aiida_cp2k/calculations/__init__.py

```diff
@@ -57,13 +57,38 @@ def define(cls, spec):
                    default=cls._DEFAULT_PARSER,
                    non_db=True)
 
+        # Add output_filename attribute
+        spec.input('metadata.options.output_filename', valid_type=six.string_types, default=cls._DEFAULT_OUTPUT_FILE)
```

Comment:
Contrary to what has been done in the PR https://github.com/aiidateam/aiida-cp2k/pull/43 following the issue #21, here I moved the `output_filename` option back. The reason for that is not to make the output filename modifiable, but to have the output filename appear in the attributes so that one can do:
```
content_string = calc.outputs.retrieved.get_object_content(calc.get_attribute('output_filename'))
```